admin-dev/themes/new-theme/js/pages/product/components/dropzone/Dropzone.vue

```diff
@@ -87,9 +87,10 @@
       :confirmation="true"
       :modal-title="
         $tc('modal.title', this.selectedFiles.length, {
-          '%filesNb%': this.selectedFiles.length,
+          '{filesNb}': this.selectedFiles.length,
```

Comment:
Already commented below, but I'm not sure we should change the wording format unless the `%` separator is really not compatible anymore If it's just about some convention I'm not sure it's enough of an argument to change the existing wording